src/OpenFeature/Providers/Memory/InMemoryProvider.cs

```diff
@@ -0,0 +1,129 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using OpenFeature.Constant;
+using OpenFeature.Error;
+using OpenFeature.Model;
+
```

Comment:
```suggestion

#nullable enable
```

Let's force the nullable validations/checks in the new class files so it would be easier later on to enable for the entire project.